SimG4Core/MagneticField/interface/Field.h

```diff
@@ -3,14 +3,18 @@
 
 #include "G4MagneticField.hh"
 
-class MagneticField;
+#include "MagneticField/Engine/interface/MagneticField.h"
+
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
Please, replace the header by 

#include <CLHEP/Units/SystemOfUnits.h>

this header should be used inside header files.